example/deployment/bash-scripts/setScriptPath.sh

```diff
@@ -1,9 +1,8 @@
-#!/usr/bin/env bash
+#!/bin/bash
 # sets up the different paths for calling deployment scripts
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-# Find example directory and normalise
-export EXAMPLE="$(cd "$DIR/../.." >/dev/null && pwd -P)"
+export EXAMPLE="./example"
```

Comment:
Is this taking a step backwards? My understanding is that the previous code allowed the script to work regardless as to where the user is running the scripts from, instead of requiring them to be run from the Palisade root directory.

Reply:
correct, this shouldn't be changed like that.